stomp_ws/client.py

```diff
@@ -42,21 +42,21 @@ def _connect(self, timeout=0):
             if 0 < timeout < total_ms:
                 raise TimeoutError(f"Connection to {self.url} timed out")
 
-    def _on_open(self):
+    def _on_open(self, ws):
         self.opened = True
 
-    def _on_close(self):
+    def _on_close(self, ws):
         self.connected = False
         logging.debug("Whoops! Lost connection to " + self.ws.url)
         self._clean_up()
 
-    def _on_error(self, error):
+    def _on_error(self, ws):
```

Comment:
error not available on line below?

Reply:
Ha, side effect of trying to fix this twice.

Reply:
was fixed